docs/installation.md

```diff
@@ -92,14 +89,17 @@ If you don't want to use this command and set the key yourself, this string shou
 **If the application key is not set, your user sessions and other encrypted data will not be secure!**
 
 #### Additional Configuration
-Lychee needs almost no other configuration out of the box. You are free to get started developing! However, you may wish to review the `config/app.php` file and its documentation. It contains some options that aren't included from `.env` that you may wish to change according to your installation.
 
 You may also want to configure a few additional components of Lychee in `.env`, such as:
 
 - Cache
 - Database
 - Session
 
+They are documented in the file if you copied the `.env.example` file.
+
+Some advanced options are not defined there by default, but you can take a look at `config/app.php` to find such options. Usually, you don't have to change them and can go with the defaults. **If you don't know what they're doing, do not change them.**
```

Comment:
Nice!

However, I wonder why it is restricted to `config/app.php`? The file `app.php` is probably the last one which needs adjustments. Other files like `database.php` and `mailer.php` are much more likely. So I would write it like "take a look into the `config/` directory.

Reply:
I'm very in favour of complete examples (ideally with the default values - I suggested changing those to match elsewhere).

As a user, having to copy every directive is a pain, and then you'd probably want to add context in case you need to edit later.